evap/static/fonts/glyphicons-halflings-regular.svg

```diff
@@ -1,288 +0,0 @@
-<?xml version="1.0" standalone="no"?>
```

Comment:
just saw there's a `Sorting icons.psd` in the images directory, could you check whether we need that? also, there's the old fsr-logo in there.

Reply:
wow. very stuff. such useless.